src/pages/home/report/ReportActionsView.js

```diff
@@ -285,7 +285,7 @@ class ReportActionsView extends React.Component {
      */
     scrollToListBottom() {
         if (this.actionListElement) {
-            this.actionListElement.scrollToIndex({animated: false, index: 0});
+            this.actionListElement.scrollToOffset({animated: false, offset: 0});
```

Comment:
Yo, I definitely like the fix, but do you mind removing it from this issue and making a new one to report the problem, and propose this solution there? I know it's more effort on your part (sorry about that) but it's nice to keep each fix linked to the specific issue & investigation.